util-hadoop/src/main/java/com/google/cloud/hadoop/util/HadoopCredentialConfiguration.java

```diff
@@ -99,13 +105,4 @@ public Configuration getConf() {
     getConfigurationInto(new ConfigurationEntriesAdapter(configuration));
     return configuration;
   }
-
-  /**
-   * Load configuration values from the provided Configuration source. For any key that does not
-   * have a corresponding value in the Configuration, no changes will be made to the state of this
-   * object.
-   */
-  public void setConf(Configuration entries) {
```

Comment:
Why this method was removed?

Reply:
Never used. The patch also removed "Configurable" from the interface. The Configuration seems to be set via the constructor.